server/internal/cards/internal/repository/db/db.go

```diff
@@ -63,6 +64,49 @@ func (r *Repo) PickFieldCards(ctx context.Context, gameID uuid.UUID, playerID in
 	return nil
 }
 
+func (r *Repo) ReplenishFieldCards(ctx context.Context, gameID uuid.UUID, number int) error {
+	for i := 0; i < number; i++ {
```

Comment:
range を使った方がミスを産みにくいと思います
```suggestion
	for i := range number {
```